MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -36,8 +36,15 @@ open class BaseInstructionsBannerView: UIControl {
             
             if let distance = distance {
                 let distanceString = distanceFormatter.string(from: distance)
-                let distanceUnit = distanceFormatter.unitString(fromValue: distance, unit: distanceFormatter.unit)
-                guard let unitRange = distanceString.range(of: distanceUnit) else { return }
+                var distanceUnit = distanceFormatter.unitString(fromValue: distance, unit: distanceFormatter.unit)
+                var unitRange = distanceString.range(of: distanceUnit)
+                // In Hebrew, “meters” looks different in the distance string than it does on its own. <rdar://problem/38028709>
+                // Fall back on splitting the string on whitespace.
+                if unitRange == nil, let spaceRange = distanceString.rangeOfCharacter(from: .whitespaces) {
+                    unitRange = spaceRange.upperBound..<distanceString.endIndex
+                    distanceUnit = String(distanceString[unitRange!])
```

Comment:
if let instead of force unwrapping.

Reply:
`unitRange` is set to a non-nil value in the line right above this line. `if let` isn’t suitable because `unitRange` is used outside the `if` statement. Specifically, if `distanceString` doesn’t contain the unit string _and_ doesn’t contain a space, we still need to bail unless we come up with a different way to deemphasize the unit string. (Or should we just treat the entire string as the number?)